modules/users/server/routes/admin.server.routes.js

```diff
@@ -4,22 +4,28 @@
  * Module dependencies
  */
 var adminPolicy = require('../policies/admin.server.policy'),
-  admin = require('../controllers/admin.server.controller');
+  admin = require('../controllers/admin.server.controller'),
+  passport = require('passport'),
+  express = require('express');
 
 module.exports = function (app) {
-  // User route registration first. Ref: #713
-  require('./users.server.routes.js')(app);
+  var router = express.Router();
 
-  // Users collection routes
-  app.route('/api/users')
+  //Set JWT Auth for all Admin Routes
+  router.all('*', passport.authenticate('jwt', { session: false }));
+
+  router.route('/')
     .get(adminPolicy.isAllowed, admin.list);
 
   // Single user routes
-  app.route('/api/users/:userId')
+  router.route('/:userId')
     .get(adminPolicy.isAllowed, admin.read)
     .put(adminPolicy.isAllowed, admin.update)
     .delete(adminPolicy.isAllowed, admin.delete);
 
   // Finish by binding the user middleware
-  app.param('userId', admin.userByID);
+  router.param('userId', admin.userByID);
+
+  app.use('/api/admin/users', router);
```

Comment:
@mleanos Its a better implementation.  Keeps the routes confined and off the main application to reduce collisions.  For instance if you wanted to do .param(:userid anywhere else it would no longer conflict like it does now.  For instance the admin routes would want to return a full user, while if you are not admin you may want to only return certain fields.  Also allows things like:

router.all('*', passport.authenticate('jwt', { session: false }));

to secure all the routes instead of adding them all individually.  There was a specific reason I had to do this, but I can't remember exactly why.  But its a better implementation regardless.  If you need a new role you just add it to the correct policy....